src/disaster_vehicle.cpp

```diff
@@ -481,7 +481,7 @@ static bool DisasterTick_Aircraft(DisasterVehicle *v, uint16_t image_override, b
 		IndustryID ind = GetIndustryIndex(tile);
 		v->dest_tile = TileIndex{ind};
 
-		if (GetIndustrySpec(Industry::Get(ind)->type)->behaviour & industry_flag) {
+		if (GetIndustrySpec(Industry::Get(ind)->type)->behaviour.Any(industry_flag)) {
```

Comment:
Why not use `IndustryBehaviour` as type and use `Test` here? All callers have one bit set as far as I can see.

Reply:
Good shout